test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetRestoreCommandTest.cs

```diff
@@ -1270,7 +1270,7 @@ public void RestoreCommand_FromHttpSource()
         }
 
         [Fact]
-        public void RestoreCommand_FromProjectJson_RelativeGlobalPackagesFolder()
+        public async Task RestoreCommand_WithPackageReference_RelativeGlobalPackagesFolder()
```

Comment:
nit... I think we do this for test names?
```suggestion
        public async Task RestoreCommand_WithPackageReference_RelativeGlobalPackagesFolderAsync()
```

Reply:
I don't think we enforce it either way for test names. 

I don't think it adds too much value given that it's not a method that's called. 

I did a quick sanity check in this class in particular and we have: 

https://github.com/NuGet/NuGet.Client/blob/2b2f1d52a4d42c21ec46aa0977abf7d8ccebd957/test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetRestoreCommandTest.cs#L198

https://github.com/NuGet/NuGet.Client/blob/2b2f1d52a4d42c21ec46aa0977abf7d8ccebd957/test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetRestoreCommandTest.cs#L1917

https://github.com/NuGet/NuGet.Client/blob/dev/test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs is one of the largest test classes and it's all over the place. 

My thinking is it's bikeshedding to bother with the test names. 

Given that this was a nit comment, I'll resolve it, but let me know if you want to chat more.